src/sentiment.py

```diff
@@ -494,8 +512,8 @@ def get_twitter_users_from_file(file):
                         help="Use twitter user ids from file")
     parser.add_argument("-n", "--newsheadlines", metavar="SYMBOL",
                         help="Get news headlines instead of Twitter using stock symbol, example: TSLA")
-    parser.add_argument("--frequency", metavar="FREQUENCY", default=3600, type=int,
-                        help="How often in seconds to retrieve news headlines (default: 3600 sec)")
+    parser.add_argument("--frequency", metavar="FREQUENCY", default=sentiment_frequency, type=int,
+                        help="How often in seconds to retrieve news headlines (default: %d sec)" % sentiment_frequency)
```

Comment:
consider using f string instead of old school variable replacement, like so: `f"How often in seconds to retrieve news headlines (default: {sentiment_frequency} sec)"`